vispy/app/backends/tests/test_rfb.py

```diff
@@ -0,0 +1,5 @@
+# This currenly only tests that the backend exists and can be imported ...
+
+def test_rfb():
```

Comment:
Let's spend 10 minutes seeing if there is an easy way to test this. Maybe ipywidgets has a test fixture that we can use to fake the notebook.

Reply:
This looks complicated but I did find this for how ipywidgets does their tests: https://ipywidgets.readthedocs.io/en/latest/dev_testing.html

Reply:
Another option is to fake it by calling `handle_event()` and `get_frame()`. Basically assuming that `jupyter_rfb` works as it should, and testing that the backend implementation does what it must do. Then we can bypass the need ffor a jupyterlab/notebook instance

Reply:
Works for me. `unittest.mock` for the win (or pytest mock)